extensions/amp-list/validator-amp-list.protoascii

```diff
@@ -144,9 +144,39 @@ tags: { # amp-list-load-more
     name: "load-more-end"
     value: ""
     mandatory_oneof: "['load-more-button', 'load-more-failed', 'load-more-end', 'load-more-loading']"
-  }
+  },
 }
 
+# The button element variant allowed in amp-list-load-more.
+tags: {
+  html_format: AMP
+  html_format: AMP4ADS
+  html_format: AMP4EMAIL
+  html_format: ACTIONS
+  requires_extension: "amp-list"
+  tag_name: "BUTTON"
+  spec_name: "amp-list-load-more button[load-more-clickable]"
+  mandatory_ancestor: "AMP-LIST-LOAD-MORE"
+  attrs: {
+    name: "disabled"
```

Comment:
Are these fields bindable, like in BUTTON's vanilla spec?

Reply:
I just went with the very simple example provided in the documentation which is probably oversimplified. Cathy would know best as she worked on this. Maybe keep as is until there is a need?